api/src/main/java/jakarta/json/JsonObjectBuilder.java

```diff
@@ -276,6 +276,8 @@ default JsonObjectBuilder remove(String name) {
      * on the order in which name/value pairs are added to the object using
      * this builder.
      *
+     * This method will clear the builder.
```

Comment:
When? I mean existing text uses present tense, why this sentence needs future one?